js/app/dashboard/components/Task.js

```diff
@@ -20,7 +20,9 @@ const TaskCaption = ({ task }) => {
 
   return (
     <span>
-      <span className="mr-1">#{ task.id }</span>
+      <span className="mr-1"> #{ task.id } | { ( task.metadata.order_number && !_.isEmpty(task.metadata.order_number) )}
```

Comment:
This doesn't make any sense, the conditional returns a boolean, but it should be a string with the order number.